addons/l10n_sa_edi/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, api
+from odoo.tools.translate import _
+from odoo.exceptions import UserError
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = 'account.move.reversal'
+
+    def _prepare_default_reversal(self, move):
+        return {
+            **super(AccountMoveReversal, self)._prepare_default_reversal(move),
+            'l10n_sa_adjustment_reason': self.reason
```

Comment:
you do not want to recuperate it from the ref?  (reason is put into the ref field of the created credit/debit note after the move.name)  It seems like we are using things multiple times.

Reply:
Done